pandas/core/series.py

```diff
@@ -2076,8 +2076,7 @@ def idxmin(self, axis=0, skipna=True, *args, **kwargs):
         >>> s.idxmin(skipna=False)
         nan
         """
-        skipna = nv.validate_argmin_with_skipna(skipna, args, kwargs)
-        i = nanops.nanargmin(self._values, skipna=skipna)
+        i = self.argmin(None, skipna, *args, **kwargs)
```

Comment:
is there a reason we are passing args, kwargs? pass skipna via kwarg

Reply:
Such that it fails when user passes unexpected kwargs to it? But I deleted those in the new commit.